hack/ci/test.sh

```diff
@@ -41,6 +41,7 @@ echo "kubectl version is $(kubectl version --client)"
 rm -f go.work go.work.sum
 go work init .
 go work use applylib
+go work use ktest
```

Comment:
This, even more, suggests to me that the workspace file should really be checked into the repo (assuming, which I think is true but haven't double-checked, that all referenced modules live within the same repo). Doing that would, I believe, help with making local development easier, so that one can just clone the repo and have it work both in an editor and with other tooling (e.g. `go test`) without having to first figure out how to make the modules reference each-other in sane ways.

Reply:
Yes, my only concern is that I haven't seen it elsewhere, but given the pain of not having it checked in you have persuaded me and I think it's worth doing.  If we find something unexpected, then we can revert it.  I'll do a separate PR I guess where we do all this in one (that's the hope, anyway!)